### PR TITLE
sql-44: As a developer I want to have mechanism to connect to database from console

### PR DESCRIPTION
**Pull request by Zoranais**

### CODE REVIEW

File: backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+{
+    public class PostgreSqlService : IDatabaseService
+    {
+        private string _connectionString;
+
+        public string ConnectionString
+        {
+            get { return _connectionString; }
+            set { _connectionString = value; }
+        }
```
Comment (Anatolii-Ivanov) [inline backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs]: It's not Java 
You can just write public property

File: backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        public async Task<string> ExecuteQueryAsync(string query)
+        {
+            using NpgsqlConnection connection = new NpgsqlConnection(_connectionString);
+
+            using NpgsqlCommand command = connection.CreateCommand();
+            command.CommandText = query;
+            try
+            {
+                await connection.OpenAsync();
+                NpgsqlDataReader reader = await command.ExecuteReaderAsync();
```
Comment (Anatolii-Ivanov) [inline backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs]: var reader

File: backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            get { return _connectionString; }
+            set { _connectionString = value; }
+        }
+
+        public PostgreSqlService(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+
+        public async Task<string> ExecuteQueryAsync(string query)
```
Comment (Anatolii-Ivanov) [inline backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs]: Is it possible to return non string results?

File: backend/Squirrel.ConsoleApp/Services/SqlServerService.cs [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                reader.Close();
+
+                return result;
+            }
+            catch (Exception ex)
+            {
+                throw new DatabaseException(ex.Message);
+            }
+
+        }
```
Comment (Anatolii-Ivanov) [inline backend/Squirrel.ConsoleApp/Services/SqlServerService.cs]: Too much duplicated code, actually

File: backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs (hunk 3 above)
Comment (Anatolii-Ivanov) [reply backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs]: I mean here, that you convert a table of results to a string, why not to pass some parse function or make it generic in some way?

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by serhiiyasenev — APPROVED** [review]

**Pull request merged**